test/simulator/timestamp/call_log_manager/call_log_manager.cpp

```diff
@@ -125,15 +125,14 @@ void
 call_log_manager::call_log_commit_transaction(const json &call_log_entry)
 {
     const std::string session_id = call_log_entry["session_id"].get<std::string>();
+    session_simulator *session = get_session(session_id);
+    const std::string config = call_log_entry["input"]["config"].get<std::string>();
 
-    /* If there is a failure in commit_transaction, there is no work to do. */
-    int ret = call_log_entry["return"]["return_val"].get<int>();
-    if (ret != 0)
-        throw "Cannot commit_transaction for session_id (" + session_id +
-          ") as return value in the call log is: " + std::to_string(ret);
+    int ret = session->commit_transaction(config);
 
-    session_simulator *session = get_session(session_id);
-    session->commit_transaction();
+    int ret_expected = call_log_entry["return"]["return_val"].get<int>();
+    /* The ret value should be equal to the expected ret value. */
+    assert(ret == ret_expected);
```

Comment:
Shall we print an error message if ret is a non-zero value.

Reply:
Yeah I'll add an exception like the other call log manager methods.